### PR TITLE
Wallet updates and wallet navbar changes

### LINKED ISSUE #837: Wallet Redesign Docs Update
Find and replace old wallet docs and images :) [Wallet redesign PR](https://github.com/btcpayserver/btcpayserver/pull/2164)

Existing Wallet
- [x] Hardware wallet
- [x] Import wallet file
- [x] Manually input xpub
- [x] Scan QR code
- [x] Manually input seed

New Wallet
- [x] Hot wallet
- [x] Watch-only wallet

### PR DESCRIPTION
**Pull request by britttttk**
Close #837 

- Match wallet navbar and setup wallet page to the Wallet Setup UI screens

### CODE REVIEW

**Comment by britttttk** [comment]
image

wallet setup => WalletSetup.md
existing wallet => HWI.md
new wallet => HotWallet.md

**Review by Zaxounette — APPROVED** [review]
Nicely done !

File: docs/WalletSetup.md [hunk 1]
```diff
@@ -4,40 +4,34 @@ After [account registration](./RegisterAccount.md) and [store creation](./Create
...
-:::
-
-Importing an existing wallet's seed words is another way to create a wallet in BTCPay. Although it is not recommended, this method can save private keys on the server (in addition to the xpub). 
+By using an existing wallet, you can receive payments to the external wallet, without BTCPay Server knowing the wallet's private key. If a malicious attacker hacked your server and obtained the xpub, they could observe your transaction history, but cannot access the funds.
 
-Externally generated wallet can be used as:
-- [Existing hardware wallet](#use-an-existing-hardware-wallet)
-- [Existing software wallet](#use-an-existing-software-wallet)
+- [Connect hardware wallet (Recommended)](#use-an-existing-hardware-wallet)
+- [Import Wallet file (Recommended)](#use-an-existing-software-wallet)
```
Comment (dennisreimann) [inline docs/WalletSetup.md:19]:
```suggestion
- [Connect hardware wallet (Recommended)](#connect-hardware-wallet)
- [Import Wallet file (Recommended)](#import-wallet-file)
```

**Review by dennisreimann — APPROVED** [review]
Well done! 👍

**Comment by pavlenex** [comment]
The current structure of wallet setup is confusing IMO as it doesn't resemble the UI, at least the navigation on the left side links to random pages.

I propose that we have:
- WalletSetup.md - contains overview and links to ConnectWallet and CreateWallet.md
  - Createwallet.md - contains info about wallet creation currently present in WalletSetup.md
  - Connectwallet.md - contains info about connecting a wallet currently present in WalletSetup.md 

Pages that can be modified/wiped
- HotWallet.md as info about hot wallets seems to be present in WalletSetup.md
- HardwareWalletIntegration - wiped from the navigation menu but the page should remain there and be linked from connect a wallet.md or if we want it in navigation, it should be nested under ConnectWallet.md

**Pull request merged**